spec/src/main/doc/javase/scopescontext_se.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[contexts_se]]
+
+== Scopes and contexts in Java SE
+
+[[builtin_contexts_se]]
+
+=== Context management for built-in scopes in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<builtin_contexts>> and is also required to ensure the following rules for built-in context implementation.
+
+
+[[request_context_se]]
+
+==== Request context lifecycle in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<request_context>> and is also required to ensure the following rules.
+
+The request scope is active:
+
+* during any method invocation.
```

Comment:
No. As you said in other mail thread this is temporary for EDR1 and will be
changed with CDI-530. I let it intentionally vague to let you choose the
easiest solution in this temp impl. Now if you think it should be more
precise and have suggestion, I'm all ears...
Le mar. 23 juin 2015 à 12:55, Jozef Hartinger <notifications@github.com> a
écrit :

> In spec/src/main/doc/javase/scopescontext_se.asciidoc
> <https://github.com/cdi-spec/cdi/pull/251#discussion_r33029668>:
>
> > +[[builtin_contexts_se]]
> > +
> > +=== Context management for built-in scopes in Java SE
> > +
> > +When running in Java SE, the container must extend the rules defined in <<builtin_contexts>> and is also required to ensure the following rules for built-in context implementation.
> > +
> > +
> > +[[request_context_se]]
> > +
> > +==== Request context lifecycle in Java SE
> > +
> > +When running in Java SE, the container must extend the rules defined in <<request_context>> and is also required to ensure the following rules.
> > +
> > +The request scope is active:
> > +
> > +* during any method invocation.
>
> So request context isn't shared between method invocations? I really
> cannot tell from the proposal as the wording is very vague. Has any thought
> been given on how this is supposed to be implemented?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cdi-spec/cdi/pull/251/files#r33029668>.
>